apps/meteor/app/ui-master/server/index.js

```diff
@@ -124,16 +125,11 @@ Meteor.startup(() => {
 });
 
 const renderDynamicCssList = _.debounce(
-	Meteor.bindEnvironment(() => {
+	Meteor.bindEnvironment(async () => {
 		// const variables = RocketChat.models.Settings.findOne({_id:'theme-custom-variables'}, {fields: { value: 1}});
-		const colors = Settings.find({ _id: /theme-color-rc/i }, { fields: { value: 1, editor: 1 } })
-			.fetch()
-			.filter((color) => color && color.value);
-
-		if (!colors) {
-			return;
-		}
+		const colors = await Settings.find({ _id: /theme-color-rc/i }, { projection: { value: 1, editor: 1 } }).toArray();
 		const css = colors
+			.filter((color) => color && color.value)
```

Comment:
is it better filter colors without values or let it go to CSS with an empty value?

because if `color` has no `value`, with this `filter` there will be no `--color-name: ` into the final CSS

Reply:
valid question, but since this is the original filter, I will kindly ignore, since its not the PR scope ;)